src/assignments/project-2/index.js

```diff
@@ -4,6 +4,65 @@ import autobind from 'class-autobind'
 
 import './app.scss'
 
+const Layout = ({children}) => (
```

Comment:
Let's pull this layout into another file.

Reply:
I did this... take a look?